frontend/src/app/modules/home/issues/issues.component.html

```diff
@@ -23,7 +23,7 @@
                 </div>
             </ng-template>
 
-            <p-table [value]="issues" [paginator]="paginators" [rows]="10" [resizableColumns]="true" [autoLayout]="true">
+            <p-table [value]="issues" [paginator]="paginators" [rows]="itemsPerPage" [resizableColumns]="true" [autoLayout]="true">
```

Comment:
It is a bad idea to do this through separate field paginators, because if the array is updated through SignalR, then those fields will have to be updated as well.

Reply:
I think @VadymKolesnyk  makes sense. We don't need paginators field.

Reply:
@tatianahutii Fixed it